explore-analyze/dashboards/using.md

```diff
@@ -161,31 +161,46 @@ Filter the data within a specified range of time.
 
 ### View the panel data and requests [download-csv]
 
-**View the data in visualizations and the requests that collect the data:**
+Viewing the details of all requests used to build a visualization and the resulting data helps you confirm that the visualization is showing the right data and that the requests are performing as expected.
+
+The available actions can vary depending on the panel you're inspecting. 
+
+#### View and download the data in visualizations
+
+This action is possible for all charts created using **Lens** or {{esql}}. It is not available on other types of panels, such as **Maps** or **Vega** visualizations.
```

Comment:
```suggestion
This action is possible for all charts created using **Lens** or {{esql}}. It is not available for other types of panels, such as **Maps** or **Vega** visualizations.
```
I like  "for all charts... for other types of panels" more than "for all charts... on other types of"